src/locales/ko/mystery-encounters/dancing-lessons-dialogue.json

```
@@ -1,27 +1,27 @@
 {
-  "intro": "An {{oricorioName}} dances sadly alone, without a partner.",
-  "title": "Dancing Lessons",
-  "description": "The {{oricorioName}} doesn't seem aggressive, if anything it seems sad.\n\nMaybe it just wants someone to dance with...",
-  "query": "What will you do?",
+  "intro": "{{oricorioName}}[[가]] 외롭게 혼자 춤을 추고 있다.\n주변에 파트너도 없는 것 같다!",
+  "title": "댄스 댄스 레볼루션",
+  "description": "{{oricorioName}}[[는]] 공격적이진 않아 보입니다.\n오히려 슬퍼 보이기까지 하네요.\n\n그저 함께 춤출 누군가를\n원하고 있는지도 모릅니다….",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Battle It",
-      "tooltip": "(-) Tough Battle\n(+) Gain a Baton",
-      "selected": "The {{oricorioName}} is distraught and moves to defend itself!",
-      "boss_enraged": "The {{oricorioName}}'s fear boosted its stats!"
+      "label": "싸운다",
+      "tooltip": "(-) 어려운 배틀\n(+) 바톤 획득",
```

```suggestion
      "tooltip": "(-) 어려운 배틀\n(+) 배턴 획득",
```
9세대부터 "바톤터치"가 "배턴터치"으로 바뀌었는데, 아이템 "Baton"이 전에 어떻게 번역되었는지 확인해봐야 할 듯합니다. 만약 그게 "바톤"이라면 맞춰서 "배턴"으로 바꿔야 할 것 같습니다.

---

바톤 입니다. 나중에 그부분은 PR 부탁드려요